ragna/core/_document.py

```diff
@@ -269,3 +270,32 @@ def extract_pages(self, document: Document) -> Iterator[Page]:
         ) as document:
             for number, page in enumerate(document, 1):
                 yield Page(text=page.get_text(sort=True), number=number)
+
+
+@DOCUMENT_HANDLERS.load_if_available
+class DocxDocumentHandler(DocumentHandler):
+    """Document handler for `.docx` documents.
+
+    Note: this does *not* extract text from headers or footers.
```

Comment:
With this, we get a proper note that stands out visually to the reader.

```suggestion
    !!! note
    
        This does *not* extract text from headers or footers.
```